### PR TITLE
Title filler

### PR DESCRIPTION
**Pull request by stimms**
Merge the test cleanup one first.

### CODE REVIEW

File: src/GenFu/Resources/PersonTitles.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿Mr. 
+Mrs.
+Ms.
+Miss.
+Doctor
+Dr.
+General
+Master
+Princess
```
Comment (dpaquette) [inline src/GenFu/Resources/PersonTitles.txt]: I think we need Duke in here

Comment (stimms) [inline src/GenFu/Resources/PersonTitles.txt]: We accept pull requests.

Comment (MisterJames) [inline src/GenFu/Resources/PersonTitles.txt]: @stimms wins this one.

:shipit:

Comment (dpaquette) [inline src/GenFu/Resources/PersonTitles.txt]: boo

**Pull request merged**